CloudSQLReplicator/google/cloud/pontem/sql/replicator/util/gcp_api_util.py

```diff
@@ -81,10 +89,9 @@ def create_cloudsql_instance(database_instance_body=None,
 
   default_credentials, default_project = google.auth.default()
   default_database_intance_body = {
-    'project': default_project,
     'name': 'cloudsql-db-{}'.format(uuid.uuid4()),
     'settings': {
-      'tier': DEFAULT_TIER
+      'tier': DEFAULT_2ND_GEN_TIER
```

Comment:
This is not strictly related to this code review, but are you handling retries in this function when you call service.instances().insert(..)?

Reply:
No, but I will add retry later.